controller/src/main/java/io/pravega/controller/server/rpc/auth/TestAuthHandler.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.rpc.auth;
+
+import io.pravega.client.auth.PravegaAuthHandler;
+import java.util.Map;
+
+public class TestAuthHandler implements PravegaAuthHandler {
```

Comment:
Should this be in the unit test packages ?

Reply:
This is to ensure that it is loaded dynamically during tests.